rllib/models/experimental/configs.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass, field
+from typing import List, Callable
+import functools
+
+from ray.rllib.models.experimental.base import ModelConfig, Model
+from ray.rllib.utils.annotations import DeveloperAPI
+
+
+@DeveloperAPI
+def _framework_implemented(torch: bool = True, tf: bool = True):
+    """Decorator to check if a model was implemented in a framework.
+
+    Args:
+        torch: Whether we can build this model with torch.
+        tf: Whether we can build this model with tf.
```

Comment:
We need to change this to "tf2". Otherwise, there will be confusion with the current framework="tf" consensus (being static graph). Going forward with the RLModule API, we will no longer support tf static graph and thus should consistently use "tf2" everywhere in all RLModule code.